On Cygwin, /usr/bin == /bin && /usr/lib == /lib.  Prefer /usr.

--- origsrc/cmake-2.6.2/Modules/Platform/UnixPaths.cmake	2008-09-24 13:34:34.000000000 -0500
+++ src/cmake-2.6.2/Modules/Platform/UnixPaths.cmake	2009-01-25 20:34:38.672375000 -0600
@@ -9,7 +9,7 @@ GET_FILENAME_COMPONENT(_CMAKE_INSTALL_DI
 # search types.
 LIST(APPEND CMAKE_SYSTEM_PREFIX_PATH
   # Standard
-  / /usr /usr/local
+  /usr / /usr/local
 
   # CMake install location
   "${_CMAKE_INSTALL_DIR}"
@@ -50,7 +50,7 @@ LIST(APPEND CMAKE_SYSTEM_PROGRAM_PATH
   )
 
 LIST(APPEND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES
-  /lib /usr/lib /usr/lib32 /usr/lib64
+  /usr/lib /usr/lib32 /usr/lib64 /lib
   )
 
 # Enable use of lib64 search path variants by default.
